server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -237,6 +242,58 @@ MeasurementMNode createTimeseries(PartialPath path, TSDataType dataType, TSEncod
     }
   }
 
+  //check if sdt parameters are valid
+  private void checkSDTFormat(PartialPath path, Map<String, String> props) throws IOException {
+    String s = "Failed to create timeseries.";
+
+    if (!props.containsKey("compdev")) {
```

Comment:
Hi, something like "compdev" should not be hard-code. It's better to extract them to be a variable.